server/app/repository/UserRepository.java

```diff
@@ -250,7 +250,7 @@ public void removeTrustedIntermediaryFromGroup(long id, long accountId)
     }
   }
 
-  private Optional<Account> lookupAccount(long accountId) {
+  public Optional<Account> lookupAccount(long accountId) {
```

Comment:
See the comment in TrustedIntermediaryService. If this isn't used, I think we can change the visibility back to private.

Reply:
Done!